container/deployer.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+#set -x
+
+FORCE=0
+PREFIX="openqa"
+NETWORK="${PREFIX}_net"
+PORT="80"
+ACTION=""
+
+function wait_for_container_ready {
+  container=$1
+
+  count=30
+  while [ $count -gt 0 ]; do
```

Comment:
There are for loops and local variables in bash.

Reply:
done